### ISSUE TITLE
Spotify setup instructions edit

### ISSUE THREAD

**Issue by Hadjimina** [home-assistant/home-assistant.github.io#2482]
Heyo
So I found another section in your documentation that needs clarification.
The new spotify entry should be expanded. 
I think you should change the redirect URI instructions from 

> Finally, add a Redirect URI in the following form:
> http://<your_home_assistant_url_or_local_ip>/api/spotify

To
> Finally, add a Redirect URI in the following form:
> http://<your_home_assistant_url_or_local_ip>:<your_home_assistant_port>/api/spotify OR
> https://<your_home_assistant_url_or_local_ip>:<your_home_assistant_port>/api/spotify
> depending on wether or not you are using ssl

Just my 2 cents
Cheers

**Comment by Hadjimina** [comment]
OOOPS nevermind still not working.
Please disregard the above issue.

**Comment by Hadjimina** [comment]
Nevermind the nevermind. I got it working, however I was taken aback when I entered the URI on the spotify website as stated above, then successfully authorized spotify and then was redirected to an invalid website namely 

https://<your_home_assistant_url_or_local_ip>:<your_home_assistant_port>/api/spotify

As it turns out that's not really a problem where it redirects to as long as the url you get redirected to when you click on "authorize spotify" in your "configuration" section in home assistant contains the string uri=<your_uri_from_the_above_comment> (where the "." sign replaced with a "%A and "/" replaced with a "%2" I think, not sure with what they get replaced with).

tldr: With the setup mentioned in the first comment everything should work fine, disregard the error  you get after you have authorized spotify.

**Comment by Hadjimina** [comment]
Yeah so long story short now it has stopped working. I removed the port number and now it works again. No idea why though. If anyone comes across this in the future: try both things an see if either of them works.
